front/src/reducers/editor/selectors.ts

```diff
@@ -1,8 +1,24 @@
-import { EditorState } from 'applications/editor/tools/types';
-import { RootState } from 'reducers';
 import { makeSubSelector } from 'utils/selectors';
 
+import type { EditorState } from 'applications/editor/tools/types';
+
+import type { RootState } from 'reducers';
+import { editorSlice } from 'reducers/editor';
+import type { InfraStateType } from 'reducers/infra';
+import buildInfraStateSelectors from 'reducers/infra/selectors';
+
 export const getEditorState = (state: RootState) => state.editor;
 const makeEditorSelector = makeSubSelector<EditorState>(getEditorState);
 
 export const getEditorIssues = makeEditorSelector('issues');
+
+const selectors = {
+  ...buildInfraStateSelectors<InfraStateType>(editorSlice),
+  getEditorIssues: makeEditorSelector('issues'),
+  getEditorLayers: makeEditorSelector('editorLayers'),
+  getEditorSchema: makeEditorSelector('editorSchema'),
+};
+
+export type editorSelectorsType = typeof selectors;
```

Comment:
We use CamelCase for types: EditorSelector (you can remove the s and type)

Reply:
Done